artifacts/CAMARA_common.yaml

```diff
@@ -10,6 +10,31 @@ info:
  version: 0.4.0
 paths: {}
 components:
+ parameters:
+  IPv4-Address:

```

Comment:
Why not using same format for all schemas? If we use UpperCamelCase for schema names, it is more coherent to use them as well here, e.g. Ipv4Address

Reply:
By convention HTTP headers are Kebab-Case:  [RFC 2616, section 5.3](https://www.rfc-editor.org/rfc/rfc2616#section-5.3).  So `IPv4-Address` is the appropriate **parameter name** for an HTTP header, whilst  
"DeviceIpv4Address" is the schema that the parameter is based on (which follows CAMARA UpperCamelCase convention for schema names).

Reply:
Hi Kevin, 

Understood and OK to use kebab-case for the parameter property value. I checked with Jose, he was refering only to use UpperCamelCase for the schema name, just for aligment. Usually code generators also works better with schemas named following Camelcase. So it would read (just removing "-", for the new schemas included): 

  **IPv4Address:**
   in: header
   name: IPv4-Address
   required: false
   schema:
    $ref: "#/components/schemas/DeviceIpv4Address"
    
    
WDYT?